index.html

```diff
@@ -93,12 +93,12 @@
                         <div class="media-anons-cont">
                           <div class="field field-name-field-media-cover field-type-image field-label-hidden">
                             <div class="field-items">
-                              <div class="field-item even"><a href="/news/2021/04/22/posledstviya-akciy-protesta-iz-za-zaklyucheniya-navalnogo-hronika-chast-2/"><img typeof="foaf:Image" src="/news/2021/04/22/posledstviya-akciy-protesta-iz-za-zaklyucheniya-navalnogo-hronika-chast-2/hor_to_main.webp" alt="" class="opt-img"></a></div>
+                              <div class="field-item even"><a href="/news/hronika-akciy-protesta/hor_to_main.webp" alt="" class="opt-img"></a></div>
```

Comment:
Тут как будто ссылка с картинкой перемешались
```suggestion
                              <div class="field-item even"><a href="/news/hronika-akciy-protesta/">
                                <img typeof="foaf:Image" src="/news/hronika-akciy-protesta/hor_to_main.webp" alt="" class="opt-img">
                              </a></div>
```